single_tech_samples/fabric/fabric_ci_cd/devops/option_2/ci.yaml

```
@@ -0,0 +1,37 @@
+trigger: none
+
+pr:
+  branches:
+    include:
+    - elena_dev
+
+variables:
+  - name: pythonWorkingDir
+    value: 'src/common/ddo_transform'
```

This is not existing in the repo at the moment, maybe we can call it something different as the name can be confused with that piece of the Parking Sensor if the name is just the name of the working directory in the agent should not matter I think.
```suggestion
    value: 'src/common/ci'
```

---

I have just removed this and the other Python step, as I copied it from another CI pipeline but it's not used here